### PR TITLE
Web meeting 2021-11-03

### PR DESCRIPTION
**Pull request by lucaslcode**

### CODE REVIEW

File: docs/meetings/web/2021-11-03.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Web meeting Nov 03 2021
+
+**Present:** Aapeli, Chad, Lucas, Ricardo, William
+
+## Discussion
+
+- Translations: all (?) strings now exist in json files, but app is still using the constants. Need to move across
+
+- NextJS in progress
+  - May need to downgrade to v11 for a while if I can't get tests working on 12
```
Comment (darrenvong) [inline docs/meetings/web/2021-11-03.md:10]: What's different in v11 compared to v12?

File: docs/meetings/web/2021-11-03.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Web meeting Nov 03 2021
+
+**Present:** Aapeli, Chad, Lucas, Ricardo, William
+
+## Discussion
+
+- Translations: all (?) strings now exist in json files, but app is still using the constants. Need to move across
```
Comment (darrenvong) [inline docs/meetings/web/2021-11-03.md:7]: I'm thinking the incremental move to use the JSON files is a nice idea, but it also means we can't set up the translation components until we do the swap/check the naming of the translation key etc as discussed in my other PR, which means people who want to help us translate stuff won't really be able to help. Should we just bite the bullet and do it in large batches/all in one go? 🥲

**Review by darrenvong — APPROVED** [review]
Non blocking questions I should have had answers to if I didn't miss the meeting in the first place... 🙈

File: docs/meetings/web/2021-11-03.md (hunk 1 above)
Comment (lucaslcode) [reply docs/meetings/web/2021-11-03.md:10]: 12 uses a new rust transpiler and I'm having trouble getting the tests to run (I think there is a way to use Babel though so I'll try that first).

File: docs/meetings/web/2021-11-03.md (hunk 2 above)
Comment (lucaslcode) [reply docs/meetings/web/2021-11-03.md:7]: Yeah maybe, want to do half each?

Comment (darrenvong) [reply docs/meetings/web/2021-11-03.md:7]: Yeah sounds like a plan!

**Pull request merged**